tools/semantic-miner-cli/build_graph.py

```
@@ -84,11 +89,16 @@ def main():
     # Default input path
     if not args.input:
-        args.input = Path("artifacts") / "observe" / args.context / "canonical_events.json"
+        args.input = (
+            Path("artifacts") / "observe" / args.context / "canonical_events.json"
+        )
 
     if not args.input.exists():
         print(f"Error: Input file not found: {args.input}", file=sys.stderr)
-        print(f"Hint: Run ingest_log.py first to create canonical events.", file=sys.stderr)
+        print(
+            f"Hint: Run ingest_log.py first to create canonical events.",
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)